specification/info.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.10.0",
+  "version": "0.11.1",
```

Comment:
I think the version should be upgraded to 0.12.0 since the change is kind of breaking.

Reply:
Sorry, I didn't mean that this change was breaking, I meant that the change that causes this version change is kind of breaking (changing the possible enum values). But I see that you upgraded it from `0.10.0` to `0.11.1`. Maybe change that to `0.11.0` :small_airplane:

Reply:
If you want to change the version, it _needs_ to be `0.12.0`, because this will get committed **after** the changes that are already on master, which means that when we make a new release this needs to be `0.12.0` and we can't change the version number in the json for a previous release.

Reply:
Ok, this confusion is exactly why it needs to get updated :-)

Release v10.0 was created a while ago and it included an update to info.json to make sure the version there says "version": "0.10.0"

After that release v11.0 was created but @M4tini didn't update info.json so that still says "version": "0.10.0". If you now retrieve this package via composer you will get release v11.0 but info.json will still say 0.10.0 which is confusing.

This commit is to rectify that situation. By updating info.json to say "version": "0.11.1" I can create release v11.1.If people run a composer update it will retrieve release v11.1 which will only update the info.json to say 0.11.1 instead of 0.10.0.

The only thing this changes is matching the release number with the number in info.json. There are no other commits since the release of v0.11 at this time.

Reply:
I was aware of the situation. Currently there are changes in the master branch that will require a bump to 0.12.0, if we merge this and create a new version, we would have to make this version 0.12.0, because they are already in the master branch. So if we update the info.json, it needs to be 0.12.0 to reflect the version we are actually creating.